packages/nodes-base/nodes/Microsoft/AzureCosmosDb/helpers/errorHandler.ts

```diff
@@ -0,0 +1,97 @@
+import type {
+	IExecuteSingleFunctions,
+	IN8nHttpFullResponse,
+	INodeExecutionData,
+	JsonObject,
+} from 'n8n-workflow';
+import { jsonParse, NodeApiError } from 'n8n-workflow';
+
+import type { IErrorResponse } from './interfaces';
+
+export const ErrorMap = {
+	Container: {
+		Conflict: {
+			message: 'The specified container already exists',
+			description: "Use a unique value for 'ID' and try again",
+		},
+		NotFound: {
+			message: "The required container doesn't match any existing one",
```

Comment:
the error message is quite confusing 
```suggestion
			message: "The {the actual value passed } container doesn't match any existing one",
```